irods_specific_implementation/irods_rest_query_api_implementation.h

```diff
@@ -32,7 +32,7 @@ namespace irods::rest
         {
             try {
                 auto _query_string = _request.query().get("query").get();
-                auto _query_limit = _request.query().get("limit").get();
+                auto _query_limit = _request.query().get("limit").getOrElse("16");
```

Comment:
16?  Seems real small?

Why not 100?

Reply:
Just thinking about pagination and how common I see page size of 10 to 25 results.

Does 100 seem better?

Reply:
Let’s go with 25 and see how it feels.

Reply:
Will do.

Reply:
Done.